sdkproject/Assets/Mapbox/Unity/Utilities/HTTPRequest.cs

```diff
@@ -57,26 +55,13 @@ public void Cancel()
 
 		private IEnumerator DoRequest()
 		{
-			_request.Send();
-
-			DateTime timeout = DateTime.Now.AddSeconds(_timeout);
-			bool didTimeout = false;
-
-			while (!_request.isDone)
-			{
-				yield return null;
-				if (DateTime.Now > timeout)
-				{
-					_request.Abort();
-					didTimeout = true;
-					break;
-				}
-			}
+			yield return _request.SendWebRequest();
```

Comment:
I think this is only 2017.2+? Getting missing assembly reference in 2017.1.

Reply:
Sorry, my bad. Thought it's 2017.1 already.
Just checked Unity docs, it isn't.